Explain block confirmations on transactions page (#3267)
ethereum/ethereum-org-website: src/content/developers/docs/transactions/index.md

2. The transaction is then broadcast to the network and included in a pool with lots of other transactions.
3. A miner must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
   - You may end up waiting at this stage if the network is busy and miners aren't able to keep up. Miners will always prioritise transactions with higher `GASPRICE` because they get to keep the fees.

> 4. Your transaction is associated with a block confirmation number. Note:

Reviewer: Still just feel if we're updating this section, the naming could use some work. "Block confirmation number" sounds like a static value associated with your transaction, and more commonly this is just referred to as "confirmations" or "number of confirmations" (vs confirmation number). 

How about
`The number of "confirmations" your transaction has starts at one with the block that includes the transaction. Every subsequent block adds to the number of confirmations. Note:`

Thoughts?

Reviewer 2: Потрібно добавити місцезнаходження особи, що підтверджує певну дію, блок. Спростить можливість відстеження ланцюжка послідовності.
 Плюсом також буде ідентифікація особи.

> 4. Your transaction is associated with a block confirmation number. Note:
>   - The confirmation number increments as new blocks are created. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. 
>   - Recent blocks may be re-organised. This gives the impression that the transaction was reversed, however, the transaction is still valid but may be included in a different block.
>   - The probability of a re-organisation diminishes with every subsequent block mined, i.e. the larger the block confirmation number, the more immutable the transaction is. So, for higher value transactions, more block confirmations may be desired.

Reviewer 3: ```suggestion
Your transaction will receive "confirmations". The number of confirmations is the number of blocks created since the block that included your transaction. The higher the number, the greater the certainty that the network processed and recognised the transaction.
   - Recent blocks may get re-organised, giving the impression the transaction was unsuccessful; however, the transaction may still be valid but included in a different block.
   - The probability of a re-organisation diminishes with every subsequent block mined, i.e. the greater the number of confirmations, the more immutable the transaction is.
```

2. The transaction is then broadcast to the network and included in a pool with lots of other transactions.
3. A miner must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
   - You may end up waiting at this stage if the network is busy and miners aren't able to keep up. Miners will always prioritise transactions with higher `GASPRICE` because they get to keep the fees.

> 4. Your transaction will also get a block confirmation number. This is the number of blocks created since the block that your transaction was included in. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. This is because sometimes the block your transaction was included in may not have made it into the chain.

Reviewer 2: ???

As already said, the called contract (which can be the same as the caller) will receive a freshly cleared instance of memory and has access to the call payload - which will be provided in a separate area called the calldata. After it has finished execution, it can return data which will be stored at a location in the caller’s memory preallocated by the caller.

> Calls are limited to a depth of 1024 which means, that for more complex operations, loops should be preferred over recursive calls.

Reviewer 3: ```suggestion
Calls are limited to a depth of 1024, so you should prefer loops over recursive calls for more complex operations.
```

As already said, the called contract (which can be the same as the caller) will receive a freshly cleared instance of memory and has access to the call payload - which will be provided in a separate area called the calldata. After it has finished execution, it can return data which will be stored at a location in the caller’s memory preallocated by the caller.

> Calls are limited to a depth of 1024, which means, that for more complex operations, loops should be preferred over recursive calls.

Reviewer 3: This grammar suggestion is incorrect 

```suggestion
Calls are limited to a depth of 1024 which means, that for more complex operations, loops should be preferred over recursive calls.
```
